watchtower-plugin/src/net/http.rs

```diff
@@ -154,6 +155,31 @@ pub async fn send_appointment(
     }
 }
 
+/// Handles a ping request to the tower.
+pub async fn ping(server_url: &str) -> Result<(), String> {
```

Comment:
This should not return a `String` in case of an Error. Check what other methods are doing, there's an enum already defined for this.

Reply:
Done! I used `RequestError`